Source/Core/VideoBackends/OGL/VertexManager.h

```diff
@@ -37,9 +37,8 @@ class VertexManager : public VertexManagerBase
   void CreateDeviceObjects() override;
   void DestroyDeviceObjects() override;
 
-  // NativeVertexFormat use this
-  GLuint m_vertex_buffers;
-  GLuint m_index_buffers;
+  GLuint GetVertexBufferName() const;
```

Comment:
"Name" makes me thing it would be a string. Either just "GetVertexBuffer", or something like "Id" / "Handle" would be better.

Reply:
That's fair, I was just following what Khronos was documenting them as. I'll change it to 'handle', which is much more indicative.